### PR TITLE
Present placeholder organisations

### PR DESCRIPTION
**Pull request by tvararu**
Presents the organisation brand and logo information.

Probably easier to review commit by commit.

Relates to https://github.com/alphagov/govuk-content-schemas/pull/262.

https://trello.com/c/cJGrtnc1

### CODE REVIEW

File: app/presenters/publishing_api_presenters/organisation.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require_relative '../publishing_api_presenters'
+
+class PublishingApiPresenters::Organisation < PublishingApiPresenters::Placeholder
+  include ApplicationHelper
+
+  def details
+    super.tap do |d|
```
Comment (boffbowsh) [inline app/presenters/publishing_api_presenters/organisation.rb]:
```ruby
super.merge(
  brand: brand,
  logo: {
    formatted_title: formatted_title,
    crest: crest,
  }
)
```
might be clearer

Comment (benlovell) [inline app/presenters/publishing_api_presenters/organisation.rb]: Yeah, that's miles nicer.

File: test/factories/organisations.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :organisation, traits: [:translated] do
     sequence(:name) { |index| "organisation-#{index}" }
-    sequence(:logo_formatted_name) { |index| "organisation-#{index} logo text".split(" ").join("\n") }
+    sequence(:logo_formatted_name) { name.split.join("\n") }
```
Comment (boffbowsh) [inline test/factories/organisations.rb]: This doesn't need to be a sequence any more

File: db/data_migration/20160309153708_republish_organisations.rb [hunk 3]
```diff
@@ -0,0 +1 @@
+Organisation.all.each(&:publish_to_publishing_api)
```
Comment (boffbowsh) [inline db/data_migration/20160309153708_republish_organisations.rb]: Should use the republisher now for this: https://github.com/alphagov/whitehall/blob/master/lib/data_hygiene/publishing_api_republisher.rb. See https://github.com/alphagov/whitehall/blob/master/db/data_migration/20160219145627_republish_working_groups.rb for an example.

File: test/unit/presenters/publishing_api_presenters/organisation_test.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      },
+      analytics_identifier: "O123",
+    }
+
+    presented_item = present(organisation)
+
+    assert_equal expected_hash, presented_item.content
+    assert_equal Hash.new, presented_item.links
+    assert_equal "major", presented_item.update_type
+    assert_equal organisation.content_id, presented_item.content_id
```
Comment (boffbowsh) [inline test/unit/presenters/publishing_api_presenters/organisation_test.rb]: Should test that it's valid against the placeholder schema too

**Comment by benlovell** [comment]
@boffbowsh Addressed, but we'll need to wait for https://github.com/alphagov/govuk-content-schemas/pull/262 to be merged before we can go :green_apple:

**Comment by tijmenb** [comment]
Made a comment on https://github.com/alphagov/govuk-content-schemas/pull/262 too.

**Comment by boffbowsh** [comment]
Merge when :green_heart:

**Comment by tvararu** [comment]
Rebased, should pass tests now that https://github.com/alphagov/govuk-content-schemas/pull/262 is merged.

**Pull request merged**